src/test/kotlin/com/terraformation/backend/tracking/model/PlantingZoneModelTest.kt

```diff
@@ -37,6 +39,40 @@ class PlantingZoneModelTest {
       }
     }
 
+    @Test
+    fun `does not choose plots that lie partially outside subzone`() {
+      val model =
+          plantingZoneModel(
+              numTemporaryPlots = 1,
+              subzones =
+                  listOf(
+                      plantingSubzoneModel(
+                          boundary = multiPolygon(polygon(0.0, 0.0, 5.0, 5.0)),
+                          id = 1,
+                          plots =
+                              listOf(
+                                  monitoringPlotModel(
+                                      boundary = polygon(0.0, 0.0, 1.0, 1.0), id = 10),
+                                  monitoringPlotModel(
+                                      boundary = polygon(4.5, 0.0, 5.5, 1.0), id = 11),
+                              )),
+                      plantingSubzoneModel(
+                          boundary = multiPolygon(polygon(5.0, 0.0, 10.0, 5.0)),
+                          id = 2,
+                          plots =
+                              listOf(
+                                  monitoringPlotModel(
+                                      boundary = polygon(6.0, 0.0, 7.0, 1.0), id = 20)))))
+
+      val expected = monitoringPlotIds(10)
+
+      repeatTest {
```

Comment:
unrelated but curious why the repeat

Reply:
The plot selection is randomized, so if we weren't correctly filtering out the plot that's not fully in the subzone, the test would still pass roughly half the time.